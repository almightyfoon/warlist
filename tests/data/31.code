c15b1b8W9E9kakfMfManagag9K9OcXgu__7D9y9Haj9b9bg8aoada5a6a9gu