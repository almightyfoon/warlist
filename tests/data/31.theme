{"allowed":[80,84,154,156,223,226,230,233,234,235,236,238,240,244,291,320,323,334,335,336,337,455,456,457,499,500,501,503,520,522,523,647,649,655,656,688,696,726,778,815,2048,2053,2066,3060,3061,3062,3067,3073,3138,3153,3178,3179,3183,3188,3189,6160,6178,6185,6186,6187,6188,6189,6190,6191,6192,6193,6302,6392,6403,6404,6405,6414],"required":[9,10,22,24,25,28,48,52,54,80,84,95,97,98,99,104,107,111,113,114,115,117,119,123,128,130,132,136,137,139,142,148,151,153,154,155,157,159,160,162,164,166,168,171,178,180,182,191,199,237,266,274,275,279,281,283,286,287,293,294,295,303,328,329,330,331,333,339,340,342,344,345,347,348,349,350,351,352,353,354,355,357,365,366,371,384,388,390,391,401,428,431,432,433,434,435,438,439,440,442,445,446,448,450,451,458,478,479,481,504,510,511,513,518,524,542,544,547,549,555,556,557,560,561,562,565,588,589,601,603,604,605,606,608,609,611,612,621,622,630,637,639,640,642,643,644,646,651,669,697,699,700,702,704,705,706,800,806,815,816,820,822,823,824,2010,2034,2035,2036,2037,2038,2039,2045,2065,2069,3003,3057,3065,3066,3067,3068,3091,3092,3096,3099,3102,3122,3136,3153,3165,3171,3174,3179,4002,4006,4007,4008,4009,4013,4023,4030,4034,4038,4039,6037,6038,6039,6040,6041,6042,6043,6052,6073,6074,6075,6094,6100,6102,6103,6122,6125,6133,6135,6138,6139,6166,6172,6177,6188,6191,6207,6208,6209,6211,6213,6240,6243,6250,6264,6274,6275,6278,6279,6289,6291,6293,6345,6356,6370,6394,6395,6396,6401,6406,6407,6408,6412,6417],"free":[1,57,124,131,134,135,138,141,143,144,145,146,147,150,156,158,163,177,181,183,185,186,187,188,189,192,193,194,195,196,197,198,202,204,205,208,211,215,216,219,222,234,238,239,240,241,244,268,276,285,291,296,297,298,299,300,312,315,358,359,360,361,362,368,369,373,374,375,376,379,381,387,399,403,404,405,407,420,421,422,423,441,452,454,459,461,462,463,464,465,467,468,482,483,484,485,486,487,488,514,515,517,519,525,526,527,563,564,567,568,569,570,571,582,613,614,615,616,617,618,629,633,653,656,672,698,774,784,802,821,2011,2012,2024,2025,2026,2027,2042,2043,2044,2048,2053,2067,3039,3058,3093,3103,3104,3134,3137,3138,3142,3154,3155,3157,3169,3175,3180,3183,3187,3188,3189,4001,4004,4010,4012,4021,4024,4027,4028,4031,4032,6030,6045,6046,6048,6049,6050,6051,6063,6066,6070,6071,6076,6098,6099,6106,6137,6140,6154,6155,6167,6168,6169,6171,6185,6186,6189,6190,6191,6193,6202,6203,6214,6215,6216,6217,6236,6265,6266,6292,6295,6296,6297,6308,6311,6369,6377,6382,6383,6389,6397,6402,6409,6410,6411,6418,6419,6420]}