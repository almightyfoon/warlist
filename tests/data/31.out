https://conflictchamber.com/#c15b1b8W9EiPakfMfManagag9KiacXgu__7D9y9Haj9b9bg8aoada5a6a9gu

Cygnar Army - 72 / 75 points

[Caine 2] Captain Allister Caine [+27]
 - Defender [16]
 - Hunter [10]
 - Squire [5]
Arcane Tempest Rifleman [4]
Arcane Tempest Rifleman [4]
Captain Arlan Strangewayes [4]
Gun Mage Captain Adept [5]
Gun Mage Captain Adept [5]
Arcane Tempest Gun Mages [11]
Long Gunner Infantry (max) [16]
Tempest Blazers (max) [18]
Fuel Cache (Objective) [0]

Cygnar Army - 75 / 75 points

[Stryker 2] Lord Commander Stryker [+28]
 - Lancer [10]
 - Stormclad [18]
Journeyman Warcaster [4]
 - Charger [9]
 - Charger [9]
Major Harrison Gibbs [4]
Major Katherine Laddermore [8]
Storm Lances (max) [20]
Stormblade Infantry [10]
 - Stormblade Infantry Officer & Standard [5]
 - Stormblade Infantry Storm Gunner (3) [6]
Fuel Cache (Objective) [0]
